crypto/cipher_extra/tls_cbc.c

```diff
@@ -336,3 +333,189 @@ int EVP_tls_cbc_digest_record(const EVP_MD *md, uint8_t *md_out,
   *md_out_size = SHA_DIGEST_LENGTH;
   return 1;
 }
+
+int EVP_sha256_final_with_secret_suffix(SHA256_CTX *ctx,
```

Comment:
Should probably had aligned with the name for `EVP_tls_cbc_digest_record_sha256()`.

Reply:
done